package.json

```diff
@@ -34,30 +38,28 @@
     "@types/eslint": "^9.6.1",
     "@types/jest": "^30.0.0",
     "@types/node": "^24.5.2",
-    "@typescript-eslint/eslint-plugin": "^8.34.1",
-    "@typescript-eslint/parser": "^8.33.0",
     "axe-core": "^4.10.3",
     "babel-jest": "^30.2.0",
     "concurrently": "^9.2.1",
-    "eslint": "^8.57.1",
-    "eslint-config-prettier": "^10.1.5",
-    "eslint-config-standard": "^17.1.0",
-    "eslint-plugin-es-x": "^8.6.2",
-    "eslint-plugin-import": "^2.31.0",
+    "eslint": "^9.37.0",
+    "eslint-config-prettier": "^10.1.8",
+    "eslint-plugin-es-x": "^9.1.0",
+    "eslint-plugin-import": "^2.32.0",
     "eslint-plugin-jest": "^29.0.1",
     "eslint-plugin-jest-dom": "^5.5.0",
-    "eslint-plugin-jsdoc": "^51.0.3",
-    "eslint-plugin-markdown": "^5.1.0",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-promise": "^6.6.0",
+    "eslint-plugin-jsdoc": "^60.8.2",
+    "eslint-plugin-n": "^17.23.1",
+    "eslint-plugin-promise": "^7.2.1",
+    "globals": "^16.4.0",
     "jest": "^30.2.0",
     "jest-environment-jsdom": "^30.2.0",
     "jsdom": "^27.0.0",
```

Comment:
Should `neostandard` be in here? I'm getting a `[ERR_MODULE_NOT_FOUND]` when I try to run this locally with `make test-lint`

Reply:
I've updated the ticket to remove the requirement on `neostandard` btw

We were using `esling-config-prettier` to unset all the style/formatting rules anyway

(Prettier was doing formatting already)

Reply:
Sorry, I did try to resurrect your old PR but it was greyed out (but yeah, same branch name!)

Reply:
Cool. I was trying to have a look at the failing tests on here. I think a rebase from main will sort those out.